packages/core-js/modules/esnext.composite-symbol.js

```diff
@@ -0,0 +1,14 @@
+var path = require('../internals/path');
+var global = require('../internals/global');
+var getCompositeKeyNode = require('../internals/composite-key');
+
+var initializer = function () {
+  return path.Symbol ? path.Symbol() : global.Symbol();
+};
+
+// https://github.com/bmeck/proposal-richer-keys/tree/master/compositeKey
+require('../internals/export')({ global: true }, {
+  compositeSymbol: function compositeSymbol() {
+    return getCompositeKeyNode.apply(null, arguments).get('symbol', initializer);
+  }
+});
```

Comment:
cc @bmeck

Reply:
compositeSymbol doesn't require a lifetime (forgot to fix a readme mismatch on first commit of that), so maybe throw an extra constant argument in front of `arguments`?

Reply:
an object constant would also need to be cross realm for that implementation to work, no?

Reply:
@ljharb the constant would need to be the same on all polyfills, not necessarily cross realm in nature since that constant can come from a specific realm and not be exposed to other realms.

Reply:
@bmeck yes but `compositeSymbol(Symbol.for('a'), Symbol.for('b'))` needs to be `===` the result of the same code ran in any realm.

Reply:
yup, any conflicting polyfills (either by being a different library or by being loaded in other realms) would need to agree on the constant

Reply:
Done.